transform/mattermost-analytics/models/marts/nps/_nps__models.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+
+models:
+  - name: dim_nps_feedback
+    description: |
+      List of known customers with active on prem licenses.
+
+    columns:
+      - name: server_id
+        description: The ID of the server the event originated from..
+        tests:
+          - not_null
+      - name: user_id
+        description: The ID of the user that sent the event.
+      - name: license_id
+        description: The license id.
+      - name: feedback
+        description: NPS feedback submitted.
+      - name: score
+        description: NPS score submitted.
```

Comment:
might also help document the possible range for scores.

Reply:
Added test for accepted values range of scores.